src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -390,6 +391,7 @@ private void addOperations(Set<String> methodSignatures) throws OperationParseEx
    *   Note: if you add to this list, also update the Javadoc for check_object_contracts.
    */
   private void addDefaultContracts() {
+    contracts.add(EqualsTransitive.getInstance());
```

Comment:
Simpler contracts should appear before later ones, to make output more comprehensible.
That will be especially true once the implementation of transitivity assumes that reflexivity and symmetry have already been checked.